js/main.js

```diff
@@ -26,6 +28,18 @@ function storeValues(event) {
   placeHolder.setAttribute('src', './images/placeholder-image-square.jpg');
   viewSwap('entries');
   toggleNoEntries();
+
+  if (data.editing !== null) {
+    for (var i = 0; i < data.entries.length; i++) {
+      if (data.editing.entryId === data.entries[i].entryId) {
+        data.entries[0].entryId = data.entries[i].entryId;
+        data.entries.splice([i], 1);
```

Comment:
You are passing an array as your first argument to `splice`, which is not valid. It is meant to be given an index number. If this is working, it is working very accidentally.